database/store.go

```diff
@@ -178,7 +170,7 @@ func (s store) GetTeams(tag string) ([]model.Team, error) {
 	return teams, nil
 }
 
-func (s store) UpdateTeamSolvedChallenge(in *pb.UpdateTeamSolvedChallengeRequest) (string, error) {
+func (s *store) UpdateTeamSolvedChallenge(in *pb.UpdateTeamSolvedChallengeRequest) (string, error) {
```

Comment:
why store is pointer now ?

Reply:
It was required to change it into  a pointer